core-bundle/contao/templates/modules/mod_navigation_accessible.html.twig

```diff
@@ -0,0 +1,96 @@
+{% trans_default_domain "contao_default" %}
+{% use "@Contao/component/_stylesheet.html.twig" %}
+
+{% set wrapper_attributes = attrs()
+    .addClass([class, 'nav-wrapper', 'block'])
+    .mergeWith(wrapper_attributes|default)
+%}
+
+{% set button_attributes = attrs()
+    .addClass('nav-burger')
+    .set('title', 'MSC.ariaMenuToggle'|trans)
+    .set('aria-haspopup', 'menu')
+    .set('aria-expanded', 'false')
+    .mergeWith(button_attributes|default)
+%}
```

Comment:
Just a nitpick: in the other templates, we define the attributes right above the element where we apply them (unless in a loop) instead of at the top of the template. Might make it easier to associate which definition belongs to which element.

Reply:
Makes sense, I generally wrote it all above in the twig surrogates I've been working with but it's fine :)

See b35ec229e19f4f217ba14c42890526a8d8d077c4